### PR TITLE
Cleanup some logger

### PR DESCRIPTION
**Pull request by pvizeli**
## Proposed change

Some of the loggers are changed to debug, but they are very usefully for bug hunting.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [ ] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

**Review by ludeeus — APPROVED** [review]

File: supervisor/homeassistant/secrets.py [hunk 1]
```diff
@@ -61,4 +61,4 @@ async def _read_secrets(self):
         except (YAMLError, AttributeError) as err:
             _LOGGER.error("Can't process Home Assistant secrets: %s", err)
         else:
-            _LOGGER.debug("Reloading Home Assistant secrets: %s", len(self.secrets))
+            _LOGGER.info("Reloading Home Assistant secrets: %s", len(self.secrets))
```
Comment (frenck) [inline supervisor/homeassistant/secrets.py:64]: Hmm, do we really want to show this every single time by default?

Comment (ludeeus) [reply supervisor/homeassistant/secrets.py:64]: This will help users not running debug see that it actually is happening.

Comment (frenck) [reply supervisor/homeassistant/secrets.py:64]: Yes, I get that, but my worry is, is that this will spam the logs.

Comment (pvizeli) [reply supervisor/homeassistant/secrets.py:64]: Well, it was over 1.5 year on info before...

Comment (frenck) [reply supervisor/homeassistant/secrets.py:64]: True, so the throttle isn't needed anymore?

**Pull request merged**